Add electron-ipc-responder

swissmanu opened the pull request:
Adds `electron-ipc-adapter` to `awesome-electron`s component section.
https://github.com/swissmanu/electron-ipc-adapter

Thank you for providing the list :)

sindresorhus commented:
It's unclear in the readme when you would want this over just using the vanilla IPC module. From the example it's just looks more verbose and with classes. The link to an example in the readme also doesn't work.

swissmanu commented:
thx for your feedback. i will give the readme some love for more clear information.

swissmanu commented:
First steps done:
* Renamed to `electron-ipc-responder` for a little bit more meaning with the name.
* Added information about ES5 to readme.

Examples are coming up next.
